qt/pool.cpp

```diff
@@ -65,9 +68,24 @@ pool_changed_cb (AsPool *cpool, AppStream::Pool *qpool)
     qpool->changed();
 }
 
+static void
+pool_ready_async_cb (AsPool *cpool, GAsyncResult *result, gpointer user_data)
+{
+    auto *d = static_cast<PoolPrivate *>(user_data);
+    g_autoptr(GError) error = NULL;
+
+    if (as_pool_load_finish (cpool, result, &error)) {
+        Q_EMIT d->q->loaded();
```

Comment:
Hmm... I wonder whether instead of having a `loaded` and `failedToLoad` signal, we should just have one `loadFinished(success)` signal instead that takes a boolean to indicate whether loading was a success or wasn't. What do you think?

Reply:
Merged them into a `loadFinished(bool)` signal.